### PR TITLE
Sprint7/ObsInspiredOxygen Changes

### PR DESCRIPTION
**Pull request by vickyjaiswal0308**
IOPS-2243 Changes

### CODE REVIEW

File: structuredefinitions/UKCore-Observation-InspiredOxygen.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
       <rank value="1" />
     </telecom>
   </contact>
-  <description value="Defines the additional constraints and extensions on the UK Core Observation profile" />
+  <description value="Defines the additional constraints and extensions on the UK Core Observation profile. It basically describes whether a patient requires oxygen or is breathing room air." />
```
Comment (APageNHS) [inline structuredefinitions/UKCore-Observation-InspiredOxygen.xml:20]: Change this to something like "Defines the additional constraints and extensions on the UK Core Observation profile, for recording if a patient is breathing room air, or inspired oxygen."

File: structuredefinitions/UKCore-Observation-InspiredOxygen.xml [hunk 2]
```diff
@@ -68,5 +68,20 @@
       <path value="Observation.value[x].code" />
       <fixedCode value="l/min" />
     </element>
+     <element id="Observation.method">
```
Comment (APageNHS) [inline structuredefinitions/UKCore-Observation-InspiredOxygen.xml:71]: This is not needed - it should be brought in from UKCore-Obserrvation

File: structuredefinitions/UKCore-Observation-InspiredOxygen.xml [hunk 3]
```diff
@@ -68,5 +68,20 @@
...
+      <path value="Observation.method" />
+      <binding>
+        <strength value="preferred" />
+        <valueSet value="http://hl7.org/fhir/ValueSet/observation-methods" />
+      </binding>
+    </element>
+    <element id="Observation.device">
```
Comment (APageNHS) [inline structuredefinitions/UKCore-Observation-InspiredOxygen.xml:78]: This is not needed - it should be brought in from UKCore-Obserrvation

**Review by APageNHS — CHANGES_REQUESTED** [review]
Interpretation, specimen, and bodysite need switching to 0..0

File: structuredefinitions/UKCore-Observation-InspiredOxygen.xml (hunk 1 above)
Comment (vickyjaiswal0308) [reply structuredefinitions/UKCore-Observation-InspiredOxygen.xml]: Done this Change

File: structuredefinitions/UKCore-Observation-InspiredOxygen.xml (hunk 2 above)
Comment (vickyjaiswal0308) [reply structuredefinitions/UKCore-Observation-InspiredOxygen.xml]: Done this change

**Comment by vickyjaiswal0308** [comment]
> Interpretation, specimen, and bodysite need switching to 0..0

Done this Change

File: structuredefinitions/UKCore-Observation-InspiredOxygen.xml (hunk 3 above)
Comment (APageNHS) [reply structuredefinitions/UKCore-Observation-InspiredOxygen.xml:71]: This modelling on Observation.device still needs to be removed

**Review by APageNHS — APPROVED** [review]

**Pull request merged**